dev.lizardbyte.app.sunshine.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+
+Name=Sunshine
+Comment=Self-hosted game stream host for Moonlight. 
+Categories=AudioVideo;
+
+Icon=dev.lizardbyte.app.sunshine
+Exec=
```

Comment:
this should be set.

Reply:
So... our upstream desktop file is currently only used for our AppImage. These fields are populated using a the configure option in cmake. Changes to upstream regarding this should be handled in the CMakeLists so the file is configured in a similar manner for flatpak.

Example:
https://github.com/LizardByte/Sunshine/blob/master/CMakeLists.txt#L85-L86